src/Faker/Provider/pl_PL/Address.php

```diff
@@ -37,7 +37,7 @@ class Address extends \Faker\Provider\Address
         'Jamajka', 'Japonia', 'Jemen', 'Jordania', 'Kambodża', 'Kamerun', 'Kanada', 'Katar', 'Kazachstan', 'Kenia',
         'Kirgistan', 'Kiribati', 'Kolumbia', 'Komory', 'Kongo', 'Korea Południowa', 'Korea Północna', 'Kostaryka',
         'Kuba', 'Kuwejt', 'Laos', 'Lesotho', 'Liban', 'Liberia', 'Libia', 'Liechtenstein', 'Litwa', 'Luksemburg',
-        'Łotwa', 'Macedonia', 'Madagaskar', 'Malawi', 'Malediwy', 'Malezja', 'Mali', 'Malta', 'Maroko', 'Mauretania',
+        'Łotwa', 'Macedonia Północna', 'Madagaskar', 'Malawi', 'Malediwy', 'Malezja', 'Mali', 'Malta', 'Maroko', 'Mauretania',
```

Comment:
Name changed in 2019. Can you provide a new source for this? The one provided in comments is not updated.

Reply:
I think the best resource would be [MFA website](https://www.gov.pl/web/dyplomacja/informacje-dla-podrozujacych) and [this list](http://ksng.gugik.gov.pl/pliki/urzedowy_wykaz_nazw_panstw2017.pdf) of commission on standardization of foreign names. Here I've learned that Swaziland got renamed to Eswanti in 2018 so I will update this as well.

BTW I've [started](https://github.com/scybulski/Faker/commit/e0f8b65b1539caf994740213d7e6037c0a9f139b) to rename Macedonia to Northern Macedonia back in February in multiple languages, maybe will continue.

Reply:
@scybulski The most recent country list would be [this](http://ksng.gugik.gov.pl/pliki/urzedowy_wykaz_nazw_panstw2019.pdf) then. We could actually rely on INTL extension for that to keep it up to date but that could only happen in v2.0.

Reply:
I would not fully agree. The latter list is maybe recent but also consists of many entities which are not countries (British Indian Ocean Territory, Tokelau, Wake Island, British Antarctic Territory and other dependencies) or are partially or unrecognized countries (Transnistria, Nagorno Karabakh (this one is not even updated to Artsakh), Northern Cyprus) and that would require a lot of work to parse this file and check the status of every entity. So I would rely mostly on the MFA list which should be consistent with countries recognized by Poland.

Reply:
I want to follow the ISO here though since we do not want to end up in political stuff

Reply:
[ISO 3166-1](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) contains a lot of territories which are not countries like British/French/US/Dutch/… dependencies (BIOT, Gibraltar, Sint Maarten), autonomous regions (Honk Kong) and some others (Antarctica).

I would suggest
* either sticking to the MFA list as this package is called `pl_PL` so it refers to the political context of Poland. It could imply other controversial positions in other countries packages like adding Abkhazia and South Osetia in `ru_RU` or North Cyprus in `tr_TR` though,
* or sticking to the list of [UN members countries](https://en.wikipedia.org/wiki/Member_states_of_the_United_Nations) which seems to me like internationally approved and recognized list of countries.

Reply:
http://publications.europa.eu/code/en/en-5000500.htm

What about this list from the European union?